### PR TITLE
Mentioning Prometheus endpoint support

### PR DESCRIPTION
**Pull request by quantranhong1999**

### CODE REVIEW

**Review by chibenwa — COMMENTED** [review]
<3

**Review by chibenwa — APPROVED** [review]

File: doc/modules/metric_exporter.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Metric exporter
 
 # Metric exporter
 
-Metric exporter collects statistics from Rspamd and feeds them to external monitoring/graphing systems. [Graphite](https://graphiteapp.org/) is the only supported backend for now.
```
Comment (fatalbanana) [inline doc/modules/metric_exporter.md:8]: It's fine to call out support for Prometheus metrics here (perhaps in a new paragraph) - but it's wrong to claim that Prometheus is supported in metric exporter.

Comment (vstakhov) [reply doc/modules/metric_exporter.md:8]: I also agree with this point: prometheus endpoint is a `pull` endpoint, whilst `metric_exporter` utilises a `push` model for all operation. We can definitely mention prometheus support, but it should just be a link to the relevant `controller` worker documentation.

File: doc/architecture/protocol.md [hunk 2]
```diff
@@ -298,3 +298,4 @@ These also accept `POST`. The below endpoints all use `GET`:
 * `/fuzzydelhash` - Deletes entries from fuzzy according to content of `Hash:` header(s)
 * `/plugins` - Returns list of plugins or plugin specific stuff
 * `/ping` - Returns just a `pong` HTTP reply (could be used for monitoring)
+* `/metrics` - Returns OpenMetrics data
```
Comment (vstakhov) [inline doc/architecture/protocol.md:301]: One neat here: maybe we should describe all counters/gauges etc that are returned by Rspamd within this endpoint? A simple copy&paste from the output comments would be enough I suppose.

**Review by vstakhov — CHANGES_REQUESTED** [review]
I'd suggest to adjust the `metric_exporter` part at least, as it might be confusing.

**Pull request merged**